JSDemos/Demos/PivotGrid/FieldPanel/description.md

```diff
@@ -1 +1,14 @@
-The PivotGrid allows its users to&nbsp;sort and filter data, or&nbsp;manage the collection of&nbsp;fields at&nbsp;runtime. All these operations can be&nbsp;performed using the field panel located at&nbsp;the top of&nbsp;the component. Click the sorting or&nbsp;filtering icon of&nbsp;one of&nbsp;the fields to&nbsp;sort or&nbsp;filter data by&nbsp;this field. Right-click the &laquo;Sales&raquo; field and select an&nbsp;item from the appeared context menu to&nbsp;change the summary type. Drag and drop the fields to&nbsp;rearrange them.
\ No newline at end of file
+A field panel is a component that displays pivot grid fields involved in summaries calculation. It consists of four field areas: column, row, data and filter. Field panel allows users to manage fields which makes this component similar to the [field chooser](https://js.devexpress.com/Demos/WidgetsGallery/Demo/PivotGrid/IntegratedFieldChooser). You can use these components simultaneously as shown in this demo.
+
+To configure a field panel, use the [fieldPanel](/Documentation/ApiReference/UI_Components/dxPivotGrid/Configuration/fieldPanel) object. Assign **true** to this object's [visible](/Documentation/ApiReference/UI_Components/dxPivotGrid/Configuration/fieldPanel/#visible) property to display the fields panel. The [allowFieldDragging](/Documentation/ApiReference/UI_Components/dxPivotGrid/Configuration/fieldPanel/#allowFieldDragging) property defines whether users can drag and drop fields in the field panel. The following properties specify the visibility of fields from various areas in the field panel:
+
+- [showColumnFields](/Documentation/ApiReference/UI_Components/dxPivotGrid/Configuration/fieldPanel/#showColumnFields)
+- [showRowFields](/Documentation/ApiReference/UI_Components/dxPivotGrid/Configuration/fieldPanel/#showRowFields)
+- [showDataFields](/Documentation/ApiReference/UI_Components/dxPivotGrid/Configuration/fieldPanel/#showDataFields)
+- [showFilterFields](/Documentation/ApiReference/UI_Components/dxPivotGrid/Configuration/fieldPanel/#showFilterFields)
+
+Use the checkboxes below the pivot grid to show or hide the corresponding fields.
+
+If the field panel is visible and the [allowSorting](/Documentation/ApiReference/UI_Components/dxPivotGrid/Configuration/#allowSorting) and [allowFiltering](/Documentation/ApiReference/UI_Components/dxPivotGrid/Configuration/#allowFiltering) properties are set to **true**, users can click icons to sort or filter pivot grid fields.
+
+This demo also shows how to use the [onContextMenuPreparing](/Documentation/ApiReference/UI_Components/dxPivotGrid/Configuration/#onContextMenuPreparing) to manage fields displayed in the field panel at runtime. Invoke that menu to hide fields, change summary types, or display field chooser.
```

Comment:
If the field panel is visible, only then users can click its tabs (changed "icons" to "tabs"). That's why omitted "the field panel is visible" and rewrote the sentence so that it referenced the demo.
The onContextMenuPreparing is used to manage field panel's context menu.
```suggestion
In this demo, we have enabled the [allowSorting](/Documentation/ApiReference/UI_Components/dxPivotGrid/Configuration/#allowSorting) and [allowFiltering](/Documentation/ApiReference/UI_Components/dxPivotGrid/Configuration/#allowFiltering) PivotGrid properties. This allows users to click field panel's tabs and their funnel icons to sort and filter displayed values, respectively.

Additionally, we have specified the [onContextMenuPreparing](/Documentation/ApiReference/UI_Components/dxPivotGrid/Configuration/#onContextMenuPreparing) handler to add new context menu fields other than "Show Field Chooser". You can now invoke the menu to hide fields and change summary types.
```